app/presenters/taxon_presenter.rb

```diff
@@ -63,4 +63,10 @@ def accordion_content
 
     accordion_items
   end
+
+private
+
+  def ordered_child_taxons
+    taxon.world_related? ? WorldTaxonomySorter.call(taxon.child_taxons) : taxon.child_taxons
```

Comment:
I like the idea of putting as much of the logic into one place. However if we do that, then in TaxonPresenter we'd always be returning the `child_taxons` via the `WorldTaxonomySorter.call` method (if I understand you correctly) which also feels a bit odd? Maybe the naming isn't 100% right here... anyway, let's discuss tomorrow